bfvmm/include/hve/arch/intel_x64/vmexit/ept_violation.h

```diff
@@ -90,7 +90,7 @@ class ept_violation_handler
     /// handlers
     ///
     using handler_delegate_t =
-        delegate<bool(vcpu *, info_t &)>;
```

Comment:
remind me again why this had to change?

Reply:
Its doesn't I forgot to change it back to the original. Should be good now